### PR TITLE
iceevent srcElement property

### PR DESCRIPTION
**Pull request by CharlesRA**
Hello.

I added the srcElement object in the event's data. This property can be found in the original version of RTCPeerConnectionIceEvent

### CODE REVIEW

File: lib/peerconnection.js [hunk 1]
```diff
@@ -49,15 +67,15 @@ function RTCPeerConnection() {
   };
 
   pc.onsignalingstatechange = function onsignalingstatechange() {
-    self.dispatchEvent({ type: 'signalingstatechange' });
+    self.dispatchEvent({ type: 'signalingstatechange', srcElement: self.getSrcElement() });
```
Comment (markandrus) [inline lib/peerconnection.js]: Hmm, why wouldn't we just pass `this` as `srcElement`?

File: lib/peerconnection.js [hunk 2]
```diff
@@ -18,6 +18,24 @@ function RTCPeerConnection() {
 
   EventTarget.call(this);
 
+  this.getSrcElement = function() {
```
Comment (markandrus) [inline lib/peerconnection.js]: Do we really need this? It's not a standard API, so I prefer not to add it.

**Review by markandrus — COMMENTED** [review]
Thanks for the contribution. I have some concerns about the implementation. Also, why is `srcElement` useful? Checking [MDN](https://developer.mozilla.org/en-US/docs/Web/API/Event/srcElement), I see this is no longer recommended. The standard property should be `target`, and so I think we should have support for that first. Moreover, `srcElement` should probably be implemented on some parent Event class that RTCPeerConnectionIceEvent extends as a property that derives from `target`. WDYT?

**Comment by CharlesRA** [comment]
Thank you for your reply. I will replace srcElement with target.

File: lib/peerconnection.js (hunk 1 above)
Comment (CharlesRA) [reply lib/peerconnection.js]: I have made the changement

File: lib/peerconnection.js (hunk 2 above)
Comment (CharlesRA) [reply lib/peerconnection.js]: Removed.

**Comment by markandrus** [comment]
Thank you!

**Comment by CharlesRA** [comment]
Thank you for the approve, any idea of the V.0.4.6 release date ?

**Comment by markandrus** [comment]
I'm working on it ATM. It should be out today or tomorrow.

**Pull request merged**